crates/turbo-tasks-fs/Cargo.toml

```diff
@@ -12,6 +12,12 @@ bench = false
 name = "mod"
 harness = false
 
+[features]
+# Enables dynamic linking (and hot reloading) of embedded files/dirs.
+# A binary built with this option **is not portable**, the directory
+# path will be embedded into the binary.
+dynamic_embed_contents = []
```

Comment:
this does nothing, as the macro is never used inside this crate, at best it's documentation, at worst misleading